public/app/plugins/panel/geomap/layers/data/markersLayer.tsx

```diff
@@ -67,19 +67,19 @@ export const markersLayer: MapLayerRegistryItem<MarkersConfig> = {
       ...options?.config,
     };
 
-    const legendProps = new ReplaySubject<MarkersLegendProps>(1);
-    let legend: ReactNode = null;
-    if (config.showLegend) {
-      legend = <ObservablePropsWrapper watch={legendProps} initialSubProps={{}} child={MarkersLegend} />;
-    }
-
     const style = await getStyleConfigState(config.style);
     const location = await getLocationMatchers(options.location);
     const source = new FrameVectorSource(location);
     const vectorLayer = new VectorLayer({
       source,
     });
 
+    const legendProps = new ReplaySubject<MarkersLegendProps>(1);
```

Comment:
IIUC, we have no change here -- remove the diff just to make it smaller

Reply:
It's because the entire block was moved, so it's seen as a change.

Reply:
I just mean, there is no reason to move it, so the diff can be smaller and only contain the one line that changed (for a reason)

This does not really matter, but when reviewing PRs it is nice to have the green/red lines all mean something :)